{
  test.withException(new TestWithExceptionCallback(){
    @Override public void doTest() throws SubscriptionRepairException, SubscriptionUserApiException {
    }
  }
,ErrorCode.SUB_REPAIR_BP_RECREATE_MISSING_AO_CREATE);
}
